h2/events.py

```diff
@@ -212,6 +267,11 @@ def __init__(self):
         #: the changed settings.
         self.changed_settings = {}
 
+    def __repr__(self):
+        return "<SettingsAcknowledged changed_settings:%s>" % (
+            self.changed_settings,
```

Comment:
I'm on the fence there. The settings you change don't actually take effect until they're acknowledged: that is to say, the change is not applied until this event is received. I can see either way, and right now I'm inclined to want to leave the name unchanged rather than change it and rev hyper-h2 to 3.0.0.